lib/shape.mli

```diff
@@ -1,13 +1,18 @@
 type shape
 type shapes = shape list
 
+type point
```

Comment:
We don't have to expose `point` and `rot` in the public API. They're meant for internal usage within `shape.ml`. We want a user to directly use `rotate` and not have to worry about how it's represented internally.